Apps/phgitlab/gitlab.json

```diff
@@ -0,0 +1,439 @@
+            "identifier": "list_users",
+            "description": "List all the users in the GitLab instance",
+            "verbose": "List all the users in the GitLab instance",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {},
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "list projects",
+            "identifier": "list_projects",
+            "description": "List all the projects in the GitLab instance",
+            "verbose": "List all the projects in the GitLab instance",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {},
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.*.name",
+                    "data_type": "string",
+                    "example_values": [
+                        "project-name"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.*.path",
+                    "data_type": "string",
+                    "example_values": [
+                        "path-to/project-name"
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "list branches",
+            "identifier": "list_branches",
+            "description": "List the branches of a GitLab project",
+            "verbose": "List the branches of a GitLab project",
```

Comment:
Remove the `verbose` since it is the same as the `description`. Or, make the `verbose` more elaborate compared to `description`

Reply:
I deleted the `verbose` in [this commit](https://github.com/peterbertel/phantom-apps/commit/8159a235f9d415edc12c6442f09df319460868e2).